functions/scripts/discoveryCheck.php

```diff
@@ -253,11 +253,11 @@
 							"subnetId"    =>$s->id,
 							"ip_addr"     =>$ip,
 							"hostname"    =>$hostname['name'],
-							"description" =>"-- autodiscovered --",
-							"note"        =>"This host was autodiscovered on ".$nowdate,
+							"description" =>_("-- autodiscovered --"),
+							"note"        =>_("This host was autodiscovered on")." ".$nowdate,
 							"lastSeen"    =>$nowdate,
 							"state"       =>"2",
-							"action"      =>"add"
+							"action"      =>_("add")
```

Comment:
Hi,
"action" shouldn't be translated. Can you please revert?

$values['action'] is checked against the string "add" in $Address->modify_address($values).
It will return "Invalid action" if translated.

```php
        /**
	 * Address modification
	 *
	 * @access public
	 * @param mixed $address
	 * @param bool $mail_changelog (default: true)
	 * @return void
	 */
	public function modify_address ($address, $mail_changelog = true) {
		# save changelog
		$this->mail_changelog  = $mail_changelog;
		# null empty values
		$address = $this->reformat_empty_array_fields ($address, null);
		# strip tags
		$address = $this->strip_input_tags ($address);
		# execute based on action
		if($address['action']=="add")			{ return $this->modify_address_add ($address); }							//create new address
		elseif($address['action']=="edit")		{ return $this->modify_address_edit ($address); }							//modify existing address
		elseif($address['action']=="delete")	{ return $this->modify_address_delete ($address); }							//delete address
		elseif($address['action']=="move")		{ return $this->modify_address_move ($address); }							//move to new subnet
		else									{ return $this->Result->show("danger", _("Invalid action"), true); }
	}
```

Reply:
Sure, added as a dedicated commit into #3233.
But then there should somehow a way to get these strings translatable.